docs/sdk-json-proposal/SDK_API_WORKFLOW.md

```diff
@@ -0,0 +1,160 @@
+# Mesh Data Plane SDK – API Workflow
+## General Workflow
+1. Create a Mesh client:
+   * `mesh_create_client()`
+1. Create a Mesh Tx or Rx connection:
+   * `mesh_create_tx_connection()`
+   * `mesh_create_rx_connection()`
+1. Get a buffer from the Mesh connection:
+   * `mesh_get_buffer()`
+   * `mesh_get_buffer_timeout()`
+1. Put the buffer to the Mesh connection:
+   * `mesh_put_buffer()`
+   * `mesh_put_buffer_timeout()`
+1. Shutdown the Mesh connection:
+   * `mesh_shutdown_connection()`
+1. Delete the Mesh connection:
+   * `mesh_delete_connection()`
+1. Delete the Mesh client:
+   * `mesh_delete_client()`
+
+## Configure Connection – Multipoint Group
+1. Define a Multipoint Group configuration in JSON format:
+```json
+"connType": "multipoint-group",
+"connection": {
+  "multipoint-group": {
+    "ipAddr": "224.0.0.1",
+    "port": "9003"
+  }
+}
+```
+
+## Configure Connection – Memif
+1. Define a memif configuration in JSON format:
+```json
+"connType": "memif",
```

Comment:
it is not consistent with description in SDK_API_DEFINITION

Reply:
Yeah, will fix it.